python/sdk/merlin/model.py

```diff
@@ -1004,6 +1004,8 @@ def deploy(self, environment_name: str = None,
         bar.stop()
 
         if endpoint.status != "running":
+            print("Model deployment failed. Reason:")
```

Comment:
I think it's better to include the error message in the `ValueError` instead of printing it.

Reply:
In fact i am not too sure personally if we really should throw the entire log to the user, though it's certainly better than the current status quo which the user must go to the console and find out what goes wrong via logging.

Reply:
One way would be truncating the log output before returning the error, for example. But then it becomes the problem on how do we truncate it, whether by fixed number of line or trying to infer which lines correspond to traceback / errors

Reply:
Alternatively, instead of a ValueError, we can have a dedicated error type, with the message as an attribute, and the caller can decide what to do with the log message.

Reply:
I see. Btw, I remember `message` was intended for a short description and the schema is limited https://github.com/gojek/merlin/blob/main/db-migrations/3_deployment_status.up.sql

Reply:
In the UI side, there is a use of `message` field here https://github.com/gojek/merlin/blob/5e08c02a9acecad0570f0a62f2011ce9a1d2803b/ui/src/version/VersionListTable.js#L217
Since this will be a long message and it's shown in a small table. I think we can avoid showing this for now by removing these part
```
                        {versionEndpoint.status === "failed" ? (
                          <EuiText size={defaultTextSize}>
                            <EuiIcon
                              type={"alert"}
                              size={defaultIconSize}
                              color="danger"
                            />
                            {versionEndpoint.message}
                          </EuiText>
                        )
```

Reply:
As for the DB schema you can add migration script to extend the column capacity